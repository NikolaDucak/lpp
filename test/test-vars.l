// var access; get val;
var a = 5;
var b = 3.14;
var c;
var d = nil;
var f = true;
var g = "true";

println(a,b,c,d,f,g);

// var access; set val;
a = "newa";
b = false;
c;
d = nil;
f = 100;
g = 3.15;

println(a,b,c,d,f,g);

// var access; get & set;
a = g;
b = f;
c = d;
d = b;
f = a;
g = g;

println(a,b,c,d,f,g);

