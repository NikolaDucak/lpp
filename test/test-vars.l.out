5
3.14
nil
nil
true
true
newa
false
nil
nil
100
3.15
3.15
100
nil
100
3.15
3.15
